docs_src/manuals/introduction/introduction.rst

```diff
@@ -50,160 +50,180 @@ computations in order to be optimal on the target hardware.
+  )
+  FetchContent_GetProperties(GridTools)
+  if(NOT GridTools_POPULATED)
+      FetchContent_Populate(GridTools)
+      add_subdirectory(${gridtools_SOURCE_DIR} ${gridtools_BINARY_DIR})
+  endif()
 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Enabling Clang-CUDA and AMD GPU Support (Experimental)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+where *<release_tag>* is the git tag of the |GT| release, e.g. ``v2.0.0``.
 
-Besides NVCC-based compilation for NVIDIA GPUs, |GT| additionally supports Clang-based compilation for NVIDIA and AMD GPUs.
-Due to a bug in Clang (https://bugs.llvm.org/show_bug.cgi?id=42061), this is only possible when not using OpenMP.
-As all non-GPU backends of GridTools currently require OpenMP, they have to be disabled.
-Then, Clang-based CUDA compilation can be activated by setting the CMake-variable ``GT_CUDA_COMPILATION_TYPE`` to ``Clang-CUDA``
-and setting Clang as the main C++ compiler. A fully working command for configuring GridTools could look like this:
+The following CMake options are available (see also :ref:`installation`).
 
-.. code-block:: shell
+  * By default, all |GT| tests are disabled. To enable building of the tests, set ``GT_BUILD_TESTING`` to ``ON``.
+    If tests are enabled, their behavior can be changed as described in :ref:`installation`.
+  * Use ``GT_CLANG_CUDA_MODE`` to select how CUDA code is compiled, see :ref:`installation`.
 
- export CXX=/path/to/clang++
- cmake -DGT_ENABLE_BACKEND_CUDA=ON \
-     -DGT_CUDA_COMPILATION_TYPE=Clang-CUDA \
-     -DGT_ENABLE_BACKEND_MC=OFF \
-     -DGT_ENABLE_BACKEND_X86=OFF \
-     -DGT_ENABLE_BACKEND_NAIVE=OFF \
-     -DCMAKE_INSTALL_PREFIX=/usr/local \
-     ..
+^^^^^^^^^^^^^^^
+Using GridTools
+^^^^^^^^^^^^^^^
 
-Further, GridTools can also be compiled for AMD GPUs using AMD’s HIP. But due to limitations of AMD’s current HCC compiler,
-the Clang-based HIP version has to be used, which is not yet officially distributed in the current HIP releases
-(but will be the successor of HCC-based HIP). Compilation of a Clang-based HIP compiler is relatively straightforward and
-documented in the `official HIP repository <https://github.com/ROCm-Developer-Tools/HIP/blob/master/INSTALL.md#hip-clang>`_.
-Otherwise, the configuration is similar to the one for Clang-CUDA. Just set ``GT_CUDA_COMPILATION_TYPE`` to ``HIPCC_AMDGPU``
-and the C++ compiler to HIPCC. For example:
+After the GridTools is made available by either ``find_package`` or ``FetchContent`` the following targets for the
+different |GT| modules are available
 
-.. code-block:: shell
+  * ``backend_naive``
 
- export CXX=/path/to/hipcc
- cmake -DGT_ENABLE_BACKEND_CUDA=ON \
-     -DGT_CUDA_COMPILATION_TYPE=HIPCC-AMDGPU \
-     -DGT_ENABLE_BACKEND_MC=OFF \
-     -DGT_ENABLE_BACKEND_X86=OFF \
-     -DGT_ENABLE_BACKEND_NAIVE=OFF \
-     -DCMAKE_INSTALL_PREFIX=/usr/local \
-     ..
+If *OpenMP* is available
 
-.. note::
+  * ``backend_cpu_ifirst``
```

Comment:
Update with the final decision. Do we need `ifirst` and `kfirst` separately? They both have the same dependencies. (Currently we have mc and x86).

Reply:
For me a single `cpu` would be enough…

Reply:
The decision was to keep it since then there is a 1-to-1 mapping of backends and cmake targets etc.
The interesting question will be to how to integrate a different thread pool...